generator/operation_test.go

```diff
@@ -581,7 +581,7 @@ func TestGenServerIssue890_ValidationTrueFlatteningTrue(t *testing.T) {
 func TestGenClientIssue890_ValidationTrueFlatteningTrue(t *testing.T) {
 	defer func() {
 		dr, _ := os.Getwd()
-		os.RemoveAll(dr+"/restapi/")
+		os.RemoveAll(dr + "/restapi/")
```

Comment:
we should probably use `os.RemoveAll(filepath.Join(filepath.FromSlash(dr), "restapi"))` here

Reply:
Yes. My change just came from go fmt... But I'll do it fine.